qlib/workflow/expm.py

```diff
@@ -351,6 +351,15 @@ def _get_exp(self, experiment_id=None, experiment_name=None):
             experiment_id is not None or experiment_name is not None
         ), "Please input at least one of experiment/recorder id or name before retrieving experiment/recorder."
         if experiment_id is not None:
+            try:
```

Comment:
I feel like this is a little bit redundant? If the user inputs an invalid experiment id, then mlflow will not find the corresponding experiment and we will raise an `MlflowException` in the following codes (under the `except` [here](https://github.com/microsoft/qlib/blob/d4b36bdab448859f04b163b89843594660b3323f/qlib/workflow/expm.py#L369)).

Instead, we could update the `MlflowException` error messages, which tells the user that the id of an experiment should be `int` or `str` of int.

Reply:
Maybe we could try to add `*` in the method to force users to specify the correct arguments when calling `get_exp`.